services/provider/server/server_test.go

```diff
@@ -43,7 +43,7 @@ var mockExtR = map[string]*externalResource{
 		Name: "rook-ceph-mon",
 		Kind: "Secret",
 		Data: map[string]string{
-			"ceph-username": "client.cephclient-health-checker-consumer",
+			"ceph-username": "client.7c1507bffc3030e301cdd0f9b6ecd56e",
```

Comment:
are you sure we need this specific hash or string? This might be a case only for one consumer.

Reply:
@iamniting this is test code, it is ok to assume a specific result

Reply:
yes it is unique for one consumer as it is unit test so I used specific hash